src/mocks/server_request.ts

```diff
@@ -1,11 +1,25 @@
 import { BufReader, ServerRequest } from "../../deps.ts";
 
+export interface MockServerRequestOptions {
+  headers: { [key: string]: string };
+  body?: Deno.Buffer;
+  server?: any; // TODO what is server type
+}
+
+export interface MockServerRequest extends ServerRequest {
+  server?: any; // TODO what is server type
+}
+
+export interface RequestRespondOutput extends Response { // TODO
+  send: () => RequestRespondOutput;
+}
```

Comment:
maybe these should be moved to interfaces.ts.

Reply:
Left there what remains of it, because of what the plan is with `server_request`